changelog/unreleased/pull-2840

```diff
@@ -0,0 +1,7 @@
+Enhancement: Speed-up file deletion in forget, prune and rebuild-index
+
+We've made the file deletion for the commands forget, prune and rebuild-index
```

Comment:
I'd just reuse the wording from the summary line:
"We've sped up the file deletion [...]"

Reply:
changed.